pkg/pillar/cmd/zedrouter/zedrouter.go

```diff
@@ -1589,6 +1600,7 @@ func appNetworkDoInactivateUnderlayNetwork(
 
 	appIPAddr := ulStatus.AllocatedIPAddr
 	if appIPAddr != "" {
+		appNumOnUNetFree(ctx, netstatus, ulStatus.AppNumOnUNet)
```

Comment:
What if we have a static IP set from the controller? That wouldn't come from the allocator.

Reply:
Have consolidated IP Address allocation to the bitmap space for all( bridge, static and ephimeral).